### ISSUE TITLE
Unable to read UniformRange.numberOfPoints for L1V3 documents

### ISSUE THREAD

**Issue by jonrkarr** [fbergmann/libSEDML#93]
libSED-ML can write `UniformRange.numberOfPoints`. Example:
```python
import libsedml

doc = libsedml.SedDocument()

task = doc.createRepeatedTask()
task.setId('task')

range = task.createUniformRange()
range.setId('uniformRange')
range.setStart(0.)
range.setEnd(10.)
range.setNumberOfPoints(10)
range.setType('linear')

filename = 'test.sedml'
libsedml.writeSedML(doc, filename)
assert doc.getErrorLog().getNumFailsWithSeverity(libsedml.LIBSEDML_SEV_ERROR) == 0
```

The example above produces the following SED-ML
```xml
<?xml version="1.0" encoding="UTF-8"?>
<sedML xmlns="http://sed-ml.org/sed-ml/level1/version3" level="1" version="3">
  <listOfTasks>
    <repeatedTask id="task">
      <listOfRanges>
        <uniformRange id="uniformRange" start="0" end="10" numberOfPoints="10" type="linear"/>
      </listOfRanges>
    </repeatedTask>
  </listOfTasks>
</sedML>
```

However, libSED-ML cannot read `UniformRange.numberOfPoints`. Continuing with the above example,

```python
doc2 = libsedml.readSedMLFromFile(filename)
assert doc2.getErrorLog().getNumFailsWithSeverity(libsedml.LIBSEDML_SEV_ERROR) == 0, doc2.getErrorLog().toString()
```

This produces the following error
```
(22503 [Error]) A <range> object must have the required attribute 'sedml:id'. No other attributes from the SBML Level 3 SEDML namespaces are permitted on a <range> object. 
Reference: L3V1 Sedml V1 Section
 An unknown attribute has been found.
 Attribute 'numberOfPoints' is not part of the definition of an SEDML Level 1 Version 3 <uniformRange> element.
```

**Comment by jonrkarr** [comment]
libSED-ML can read `numberOfSteps`, which isn't actually part of L1V3.

For example, libSED-ML can read the document below. The only difference between this and the above document is `numberOfPoints` is replaced with `numberOfSteps`.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<sedML xmlns="http://sed-ml.org/sed-ml/level1/version3" level="1" version="3">
  <listOfTasks>
    <repeatedTask id="task">
      <listOfRanges>
        <uniformRange id="uniformRange" start="0" end="10" numberOfSteps="10" type="linear"/>
      </listOfRanges>
    </repeatedTask>
  </listOfTasks>
</sedML>
```

**Comment by fbergmann** [comment]
this must have happened, when i accepted the prs: 

https://github.com/fbergmann/libSEDML/pull/83
https://github.com/fbergmann/libSEDML/pull/90

i'll add some test cases for these.

**Comment by fbergmann** [comment]
this has been resolved in release 2.0.16

**Comment by jonrkarr** [comment]
Thanks! This allowed us to enable a few more tests in the BioSimulators test suite.
